src/components/tabs/tabs.js

```diff
@@ -114,16 +131,7 @@ const BTabButtonHelper = /*#__PURE__*/ Vue.extend({
           this.tab.localActive ? this.bvTabs.activeNavItemClass : null
         ],
         props: { disabled: this.tab.disabled },
-        attrs: {
-          role: 'tab',
-          id: this.id,
-          // Roving tab index when keynav enabled
-          tabindex: this.tabIndex,
-          'aria-selected': this.tab.localActive && !this.tab.disabled ? 'true' : 'false',
-          'aria-setsize': this.setSize,
-          'aria-posinset': this.posInSet,
-          'aria-controls': this.controls
-        },
+        attrs: linkAttrs,
```

Comment:
Just simply do:

```js
        attrs: {
          ...tab.titleLinkAttributes,
          role: 'tab',
          id: this.id,
          // Roving tab index when keynav enabled
          tabindex: this.tabIndex,
          'aria-selected': this.tab.localActive && !this.tab.disabled ? 'true' : 'false',
          'aria-setsize': this.setSize,
          'aria-posinset': this.posInSet,
          'aria-controls': this.controls
        },
```

Reply:
As object spread will skip/ignore `null` or `undefined` variables

Reply:
@tmorehouse, thank did not know about it.